crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -552,6 +552,7 @@ impl ObjectImpl<'_> {
                     owner: Some(Owner {
                         address,
                         checkpoint_viewed_at: self.0.checkpoint_viewed_at,
+                        root_version: None,
```

Comment:
Should this be the Object's `root_version`, actually?

Reply:
That's a tricky one, because if this object's owner is an address and not an object, then I don't see the use in propagating the root version (which is gonna be the object's version itself) upwards.

That SuiAddress is not going to have dynamic fields and is not an object ofc, so `root_version` doesn't seem to serve any purpose

Reply:
Yes, you're right -- originally, I was thinking about the case where the object was transferred to another object (so its owner is an `AddressOwner`, but that address points to an object, not an account address), but in that case, the rules around parent versions and bounding do not apply.

Looking at the rest of this function, the case that this comment is relevant for is actually the `Parent` owner case (this is the case where we're travelling back up the dynamic field hierarchy) -- we should be propagating the root version there, but it's kind of orthogonal to this PR, we can do that in a follow-up.